packages/snake/helloworld/apps.py

```diff
@@ -0,0 +1,88 @@
+import gspread
+import csv
+import firebase_admin
+from firebase_admin import credentials, firestore
+
+cred = credentials.Certificate("clauth.json")
+firebase_admin.initialize_app(cred)
+gc = gspread.service_account("auth.json")
+db = firestore.client()
+
+members = gc.open("All Quarters")
+leads = gc.open("All PLead Apps")
+
+member_list = members.worksheets()
+lead_list = leads.worksheets()
```

Comment:
seems like there is a lot of duplication going on here.. maybe think of a way to simply this